### PR TITLE
docs: update security contacts and policy location

### PR DESCRIPTION
**Pull request by gracenng**
**Fixes issue #**: https://github.com/in-toto/in-toto/issues/506

**Description of the changes being introduced by the pull request**: Security policy separation from the README.md, update contacts to be @SantiagoTorres 

**Please verify and check that the pull request fulfills the following
requirements**:

- [ ] The code follows the [Code Style Guidelines](https://github.com/secure-systems-lab/code-style-guidelines#code-style-guidelines)
- [ ] Tests have been added for the bug fix or new feature
- [x] Docs have been added for the bug fix or new feature

I wonder if we still need to go through steps 1-4 of setting up [here](https://docs.github.com/en/code-security/getting-started/adding-a-security-policy-to-your-repository)

### CODE REVIEW

**Review by adityasaky — APPROVED** [review]

**Comment by adityasaky** [comment]
@gracenng could you please sign off your commit for DCO?

**Comment by gracenng** [comment]
hi @adityasaky, passed DCO check now. Can you approve running workflow?

**Comment by JustinCappos** [comment]
Looks great to me.  Maybe @SantiagoTorres should approve as he should knows best if this is his key...  :)

**Review by SantiagoTorres — APPROVED** [review]

**Comment by SantiagoTorres** [comment]
@gracenng could you update this PR? thanks!

**Comment by gracenng** [comment]
HI @SantiagoTorres - just update the branch and need maintainer approval to run workflows

**Comment by adityasaky** [comment]
@gracenng could you please rebase your original commit over `develop` and discard the merge commit altogether? :smile:

**Comment by gracenng** [comment]
Reverted the merge commit, rebased on `develop` and force pushed. Is this right? @adityasaky

**Comment by adityasaky** [comment]
Unfortunately not. I think you want to reset this branch to having just 29b58a3f0eefc72e18085196b0c37dc861e08c67. Then, run `git pull --rebase <remote> develop` to fetch the latest changes. Point the remote to this repository rather than yours to ensure you get all the latest changes. Fix any conflicts that arise during the rebase process. After the rebase is successful, you should have just the one commit (the identifier will change but the changes will be the same, minus the conflicts) in the branch, and therefore in this PR. Feel free to reach out to me on Slack if you need help through any of this!

**Comment by lukpueh** [comment]
Our [git history guidelines](https://github.com/secure-systems-lab/lab-guidelines/blob/master/git-history.md) might be helpful to fix your commits. Patch looks good otherwise. Let me know if you need help.

File: SECURITY.md
Comment (lukpueh) [inline SECURITY.md:3]: I suggest to add a title to the document, otherwise it looks a bit naked.

**Comment by lukpueh** [comment]
Thanks for adding the title and fixing the commits. The history looks good now, but your second commit misses the sign-off line. You can add it with `git rebase HEAD~1 --signoff` and then force pushing.

**Comment by gracenng** [comment]
Let's see if I did it 👀

**Comment by adityasaky** [comment]
Thanks, @gracenng!

**Pull request merged**
